### ISSUE TITLE
Simplify the Config UX to better explain HOTP vs TOTP

### ISSUE THREAD

**Issue by aweingarten** [d8-contrib-modules/ga_login#7]
As a developer I would like the config screen here `admin/config/people/tfa` to do a better job explaining the difference between HOTP(one time code) vs TOTP(time based codes).  I had to do a LOT of searching to understand what they were before I realized that the default most people want is TOTP

**Comment by therealssj** [comment]
Yes, this is true. I see that in Drupal modules have these help pages where information related to the module is provided, what do you feel is better, information on the help page or directly on admin config page at `admin/config/people/tfa`

Maybe @nerdstein can provide some input as well.

**Comment by nerdstein** [comment]
i think we should do a little of everything.

First, when your selecting, TOTP should probably be the first and default option.

Second, the selection should probably be "Time based code (TOTP)" and "One time code (HOTP)".

Third, I think a little bit of documentation on the form, as help pages, and in the readme.

My ten cents...
